contrib/internal/manifests/loadprofile/cilium_cr_list.yaml

```diff
@@ -0,0 +1,37 @@
+# count defines how many runners in the group.
+count: 10
+
+# This simulates worst-case behavior for Cilium:
+# every agent hammering apiserver with stale LIST requests for CiliumIdentity and CiliumEndpoint.
+# The request rate is much, much higher than what we'd expect to see in production with
+# client-go exponential backoff configured, so if apiserver can survive this onslaught it can
+# survive anything Cilium throws at it.
+loadProfile:
+  version: 1
+  description: cilium list profile
+  spec:
+    rate: 20     # 20 req/sec * 10 runners = 200 req/sec
+    total: 12000 # run for ~10 minutes, 600 seconds * 20/sec = 12000
```

Comment:
@fuweid are `rate` and `total` per-runner or overall? When I run this test, it's completing faster than I expect, which makes me think that `total` might be the overall number of requests across all runners.

Reply:
the `total` here is just for one runner. so each runner will send 12000 requests with rate 20.

Reply:
would you show me the result for the duration?

Reply:
discussed offline and fixed in https://github.com/Azure/kperf/pull/149